app/components/Accordion/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+*
+* Accordion
+*
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.scss';
+
+class Accordion extends React.Component { // eslint-disable-line react/prefer-stateless-function
+    state = {};
+
+    toggleAccordion(key) {
+        this.setState({
+            [key]: !this.state[key],
```

Comment:
use key like `[`Accordion_${key}`]`

Reply:
so that we will know what this state value is used for when we are looking at data